src/main/java/frc/robot/turret/Turret.java

```diff
@@ -36,23 +35,22 @@ public Rotation2d getVelocity() {
         return inputs.velocity;
     }
 
+    public void setPower(double power) {
+        turret.setPower(power);
+    }
 
-    public void handleRotateToPoint(Translation2d targetPoint, Translation2d robotPosition) {
-        Translation2d normalizedRobotPosition = targetPoint.minus(robotPosition);
-        this.turret.setPosition(
-                new Rotation2d(
-                        targetPoint.minus(robotPosition).getX(),
-                        targetPoint.minus(robotPosition).getY()
-                )
-        );
+    public void handleRotateToTarget() {
```

Comment:
this function needs to get the final target angle which will go to the iturret, do the calculations in another function